ui/src/criteria/concept.tsx

```diff
@@ -274,104 +275,112 @@ function ConceptEdit(props: ConceptEditProps) {
         />
       )}
       <Loading status={classificationState}>
-        <TreeGrid
-          columns={hierarchy ? hierarchyColumns : allColumns}
-          data={data}
-          defaultExpanded={hierarchy}
-          rowCustomization={(id: TreeGridId, rowData: TreeGridRowData) => {
-            // TODO(tjennison): Make TreeGridData's type generic so we can avoid
-            // this type assertion. Also consider passing the TreeGridItem to
-            // the callback instead of the TreeGridRowData.
-            const item = data[id] as ClassificationNodeItem;
-            if (!item || item.node.grouping) {
-              return undefined;
-            }
-
-            const column = props.config.columns[nameColumnIndex];
-            const name = rowData[column.key];
-            const newItem = {
-              key: item.node.data.key,
-              name: !!name ? String(name) : "",
-            };
-
-            if (props.config.multiSelect) {
-              const index = props.data.selected.findIndex(
-                (sel) => item.node.data.key === sel.key
-              );
+        {!data.root?.children?.length ? (
+          <Empty
+            minHeight="300px"
```

Comment:
optional: maybe make `empty.png` the default?

Reply:
That doesn't really end up better because we still need to be able to specify whether the image is displayed or not with either an extra parameter or extra state for this one. I think that outweighs specifying the image in a couple places, plus we we might show different images in the future.